source/whats-new.txt

```diff
@@ -56,6 +57,44 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.5:
+
+What's New in 6.5
+-----------------
+
+The {+driver-short+} v6.5 release includes the following features:
+
+- Moved ``ObjectId`` generation to the ``pkFactory`` class for bulk write
+  operations.
+
+.. warning::
+
+   If you previously specified the ``pkFactory`` to handle bulk writes,
+   the ``_id`` fields of the documents inserted by using bulk writes may
+   be inconsistent with the behavior in this version.
```

Comment:
```suggestion
   If you previously specified that a ``pkFactory`` instance handles bulk writes,
   the ``_id`` fields of the documents inserted by using bulk writes may
   be inconsistent with the behavior in this version.
```

Reply:
Good call! Will update to clarify an instance.